modules/test/playwright/tests/frontend-data-set-views-web/pagination.spec.ts

```diff
@@ -246,30 +246,32 @@ fragmentTest.describe(
 		fragmentTest(
 			'FDS uses default pagination configuration after creating a Data Set',
 			async ({
+				dataSetFragmentPage,
 				dataSetManagerApiHelpers,
-				fdsFragmentPage,
 				layout,
 				page,
 			}) => {
 				await fragmentTest.step('Create table fields', async () => {
-					await dataSetManagerApiHelpers.createDataSetField({
+					await dataSetManagerApiHelpers.createDataSetTableSection({
 						label_i18n: {en_US: 'Label'},
 						name: 'id',
-						r_fdsViewFDSFieldRelationship_c_fdsViewERC: dataSetERC,
+						r_dsmDataSetTableSectionRelationship_c_dsmDataSetERC:
```

Comment:
At some point we should extract these hardcoded relations into constants object, like we have in DSM.

Reply:
would it be too ugly to import the original constants file here, right?

Reply:
I don't think we don't do this anywhere in Playwright, import local exports. It might be possible, but I'm not sure. We can look into it, but we can do it in a separate task.